src/patternfly/components/Chip/chip.scss

```diff
@@ -1,54 +1,42 @@
 .pf-c-chip {
+  --pf-c-chip--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-chip--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip--PaddingBottom: var(--pf-global--spacer--form-element);
   --pf-c-chip--PaddingLeft: var(--pf-global--spacer--sm);
   --pf-c-chip--BackgroundColor: var(--pf-global--Color--light-100);
-  --pf-c-chip--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--BorderRadius: var(--pf-global--BorderRadius--sm);
-
-  // overflow chip
-  --pf-c-chip--m-overflow--BackgroundColor: var(--pf-global--BackgroundColor--200);
-  --pf-c-chip--m-overflow--PaddingLeft: 0;
-  --pf-c-chip--m-overflow--BorderWidth: 0;
+  --pf-c-chip--before--BorderColor: var(--pf-global--BorderColor--300);
+  --pf-c-chip--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-chip--before--BorderRadius: var(--pf-global--BorderRadius--sm);
 
   // overflow chip button
-  --pf-c-chip--m-overflow--c-button--BorderRadius: var(--pf-global--BorderRadius--sm);
-  --pf-c-chip--m-overflow--c-button--BorderWidth: 0;
-  --pf-c-chip--m-overflow--c-button--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-overflow--c-button--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-overflow--c-button--hover--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--c-button--hover--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--m-overflow--c-button--active--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--c-button--active--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--m-overflow--c-button--focus--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--c-button--focus--BorderColor: var(--pf-global--secondary-color--100);
-
-  // read-only chip
-  --pf-c-chip--m-read-only--PaddingTop: var(--pf-global--spacer--form-element);
-  --pf-c-chip--m-read-only--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-read-only--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-chip--m-overflow--hover--before--BorderColor: var(--pf-global--BorderColor--200);
+  --pf-c-chip--m-overflow--active--before--BorderColor: var(--pf-global--BorderColor--200);
+  --pf-c-chip--m-overflow--focus--before--BorderColor: var(--pf-global--BorderColor--200);
 
   // chip text
   --pf-c-chip__text--FontSize: var(--pf-global--FontSize--xs);
   --pf-c-chip__text--Color: var(--pf-global--Color--100);
   --pf-c-chip__text--MaxWidth: #{pf-size-prem(120px)};
```

Comment:
Maybe outside the scope of this issue, but what do you think about changing `--pf-c-chip__text--MaxWidth` to a character width, say `20ch` and/or making that a global variable? 

Reason being that if the font-size of `pf-c-chip__text` is increased, the string length doesn't adjust to that change and results in this. 
![Screen Shot 2020-04-13 at 4 11 24 PM](https://user-images.githubusercontent.com/5385435/79157275-52782180-7da2-11ea-8281-380a0b160c60.png)

Where a `ch` value would result in this
![Screen Shot 2020-04-13 at 4 11 37 PM](https://user-images.githubusercontent.com/5385435/79157597-dfbb7600-7da2-11ea-8cbd-23a25f0545a1.png)

Reply:
That's a good point, i think its outside of the scope of this issue and it might be a blocker if we need to get design involved. it also might be worth taking a look at how this is handled in all the components. Thoughts? Want me to open an issue or do you want to? @mattnolting

Reply:
Wfm! Captured the convo here: https://github.com/patternfly/patternfly/issues/2954